docs/index.rst

```diff
@@ -237,4 +237,4 @@ Cloud Storage
   client = storage.Client()
   bucket = client.get_bucket('<your-bucket-name>')
   blob = bucket.blob('my-test-file.txt')
-  blob.upload_from_string('this is test content!')
\ No newline at end of file
+  blob.upload_from_string('this is test content!')
```

Comment:
Yeah there was a missing one. I went through and made sure all whitespace issues were resolved.